### DISCUSSION TITLE
Aware of this bypass?

### DISCUSSION THREAD

**Discussion by renekal** [SafeExamBrowser/seb-win-refactoring#757] [Q&A]
https://github.com/nxvvvv/safe-exam-browser-bypass

**Comment by dbuechel** [comment]
Yes, SEB has various security measures which can detect that manipulation (among them the Browser Exam Key [BEK] and App Signature Key [ASK, integrated with SEB-Server]). The most simple and straightforward way of detecting and discouraging students from manipulating SEB is by using the SEB Verificator: https://github.com/SafeExamBrowser/seb-win-verificator.
